reflection_refraction.py

```diff
@@ -22,7 +23,7 @@ def check_alfa(alfa):
         print("\nAlfa can't be equal or more than 90 and can't be equal or less than 0.\n")
         exit()
 
-def check_index_of_refr(n1, n2):
+def check_index_of_refr(n1: float, n2: float):
```

Comment:
If your function does not return anything you can either use `-> None:` or `NoReturn`.

I use the `None` version.

Check [this comment here](https://stackoverflow.com/a/64063782).